src/Symfony/Component/Config/composer.json

```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/filesystem": "~2.8|~3.0"
+        "symfony/filesystem": "~2.8|~3.0",
+        "symfony/dependency-injection": "~3.2"
```

Comment:
That's true, but it will then lead to errors when you decide to use the pass and didn't update the DI version before.